sdcm/cluster.py

```diff
@@ -468,6 +469,18 @@ def system_log(self):
     def continuous_events_registry(self) -> ContinuousEventsRegistry:
         return self._continuous_events_registry
 
+    @property
+    def cassandra_stress_version(self):
+        if not self._cassandra_stress_version:
+            result = self.remoter.run(cmd="cassandra-stress version", ignore_status=True, verbose=True)
+            match = re.search("Version: (.*)", result.stdout)
+            if match:
+                self._cassandra_stress_version = match.group(1)
+            else:
+                self.log.error("C-S version not found!")
```

Comment:
add `result.stdout` to error message for quicker issue investigation.

Reply:
I need to remove this, it's not actually being used, it's back cause it's part of a revert I'm doing in this PR

the revert is to return the code that install c-s on-top of the loaders, (it was remove as part of switching to docker based loaders)

Reply:
I'll remove it from the revert